restore-from-backup-local.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
```

Comment:
Can we just use the existing local script and add an argument to also do a restore when using it?

Reply:
Now there's a lot of code duplication.